.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "es2015"
+    ["es2015"]
```

Comment:
Why do we have to do this?

Reply:
While I was facing a bug I added a gulp preset, but later found out that wasn't the problem, I took it out but forgot to take-out the square braces